src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/funnels-evaluate-data-series-related-events.mdx

```diff
@@ -118,14 +118,14 @@ Here's a detailed breakdown of technical details and constraints for funnel quer
 
 ## Funnel query example [#example]
 
-This example queries the browser monitoring [`PageView`](/attribute-dictionary/?event=PageView) event and its attributes. It queries unique browser sessions that have progressed from registration to account upgrades. Labels are included for each step, indicated by the keyword `AS`.
+This example queries the browser monitoring [`PageView`](/attribute-dictionary/?event=PageView) event and its attributes. It queries unique browser sessions that have progressed from browsing a product, to adding to cart, to checkout. Labels are included for each step, indicated by the keyword `AS`.
 
-```
+```sql
 FROM PageView SELECT funnel(session, 
-    WHERE pageUrl='http://www.storefront.com/signup' AS 'Email Sign Up', 
-    WHERE pageUrl='http://www.appproduction.com/basic_feature' AS 'Feature Compare', 
-    WHERE pageUrl='http://www.appproduction.com/advanced_feature/upgrade' AS 'Upgraded Account') 
-    SINCE 12 hours ago
+    WHERE pageUrl LIKE '%browse%' as 'Browse', 
+    WHERE pageUrl LIKE '%cart%' as 'Cart', 
+    WHERE pageUrl LIKE '%checkout%' as 'Checkout') 
+    SINCE 1 week AGO
```

Comment:
I guess the result will be the same, but do we want `since` and `ago` to be in lower case to match the screenshot?